.nf-core.yml

```diff
@@ -1,4 +1 @@
-files_exist:
-  - .github/markdownlint.yml
-  - bin/markdown_to_html.py
-actions_awsfulltest: False
+repository_type: pipeline
```

Comment:
```suggestion
repository_type: pipeline

lint:
  files_exist:
    - conf/igenomes.config
```
This would ignore
> Test warnings:
>     [files_exist](https://nf-co.re/tools-docs/lint_tests/files_exist.html) - File not found: conf/igenomes.config